.github/workflows/ci.yml

```diff
@@ -26,8 +26,6 @@ jobs:
         image:
           - manylinux2014-x64
           - manylinux2014-x86
-          - manylinux1-x64
-          - manylinux1-x86
           - al2-x64
```

Comment:
why are you removing manylinux1 from CI? I thought we were trying to ensure it keeps working?

Reply:
I added it but it was not working. I tried removing it temporarily, however, this fix is not working for now.